### PR TITLE
Create Sort Ingredients Command

### LINKED ISSUE #85: Feature to sort ingredientList
Similar to sorting of recipeList

### PR DESCRIPTION
**Pull request by KSanjith**
This PR resolves #85 . It does the following:
- Allow user to sort ingredients by name
  - using "sortI" command
- Prevent duplicate named ingredients from being added
- Duplicate check ignores capitalizations for both ingredients and recipes
  - e.g. "Apple Soup" and "appLe SOUP" will be counted as duplicate names

### CODE REVIEW

File: src/main/java/ymfc/commands/SortIngredientsCommand.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    public SortIngredientsCommand() {
+        super();
+    }
+
+
+    public void execute(RecipeList recipes, IngredientList ingredients, Ui ui, Storage storage) {
+        logger.log(Level.FINEST, "Executing SortIngredientsCommand");
+
+        ingredients.sortAlphabetically();
```
Comment (Sukkaito) [inline src/main/java/ymfc/commands/SortIngredientsCommand.java:27]: I don't see the new `sortAlphabetically()` method in your changes. Was it previously added by someone before?

**Review by Sukkaito — APPROVED** [review]
LGTM! 👍

File: src/main/java/ymfc/commands/SortIngredientsCommand.java (hunk 1 above)
Comment (KSanjith) [reply src/main/java/ymfc/commands/SortIngredientsCommand.java:27]: It was already copied over by @gskang-22 from my sort recipes function, back when he was setting up the ingredients class.

**Review by 3CCLY — APPROVED** [review]
LGTM

**Pull request merged**